js-webapp/README.md

```diff
@@ -9,40 +9,19 @@ _Wouldn't it be great if you could run your existing C/C++ code on the web with
 That way, loads more people would be able to see your results, interact with your algorithm, and apply it for their own
 purposes.
 
-In this blog, we'll show you how to take a simple algorithm written in C++ and make it available as a simple web
+In this blog, we'll show you how to take a simple algorithm written in C++ and make it available as a web
 application. Subsequent blogs in this series will expand on the current one by laying out more advanced topics,
 specifically how to [make the app interactive](anotherblog), how to [visualize the results](someotherblog), and
 how to [deal with long running tasks](yetanotherblog).
 
-So today's aim is to have a simple web app like the code snippet below. At first glance this is just some boilerplate to
-be able to use the ``newtonraphson.js`` library, but **the neat part is that the complete Newton-Raphson code is written
-in C++, not JavaScript**. With some trickery, we'll be able to use that C++ code from the browser, without the need to
-port it first!
+So today's aim is to have a simple web app that determines the _root_ of a mathematical function _x^3 - x^2 + 2_, i.e. the
+value of _x_ where _y = 0_. For this, we'll use the [_Newton-Raphson_ root finding method](https://www.youtube.com/watch?v=cOmAk82cr9M). Remember Newton? Quiet
+fellow, fabulous hair? Yes, _that_ Newton. Anyway, the algorithm is written C++, but **with some trickery, we'll be able
+to use that C++ code from the browser, without the need to port it first**! 
 
-```html
-<!doctype html>
-<html lang="en">
-   <head>
-      <title>Newton-Raphson</title>
-      <meta charset="utf-8">
-      <script type="text/javascript" src="newtonraphson.js"></script>
-      <script>
-         createModule().then((module) => {
-            const tolerance = 0.001;
-            const newtonraphson = new module.NewtonRaphson(tolerance);
-            const initial_guess = -20;
-            const root = newtonraphson.solve(initial_guess);
-            document.getElementById("answer")
-               .innerHTML = "Function root is approximately at x = " +
-                            root.toFixed(2);
-         });
-      </script>
-   </head>
-   <body>
-      <span id="answer"> </span>
-   </body>
-</html>
-```
+![newton.jpg](newton.jpg)
```

Comment:
Don't forget to add attribution info for images.

Reply:
yep that's why I added the <filename>.txt files in the repo so I know where they come from.